tests/test_cli.py

```diff
@@ -175,3 +174,22 @@ def test_unused_found_because_excluded_but_not_installed(  # noqa: PLR0913
 
     assert captured.out.splitlines() == expected_unused_packages
     assert exit_code == expected_exit_code
+
+
+def test_load_defaults_no_file(tmp_path):
+    missing_pyproject = tmp_path / "pyproject.toml"
+    config = cli.load_defaults(missing_pyproject)
```

Comment:
```suggestion
    empty_pyproject = tmp_path / "pyproject.toml"
    config = cli.load_defaults(empty_pyproject)
```

Reply:
it's actually missing in this test case, however I just added more tests handling the other main scenarios.

Reply:
addressed in 23a9f682c6fcd9c8ffd89e